.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
is that correct that we have just master and not feature branches like in [TinyMCE](https://github.com/tinymce/tinymce/pull/8307/files)?

Reply:
This is for PR targets, I believe, the feature branches don't need scanning until a PR is opened